### ISSUE TITLE
Ивент TeamVsTeam (TvT)

### ISSUE THREAD

**Issue by noargsconstruct** [goodwinxp/Yorozuya#82]
Давно была идея, хотелось бы попробовать реализовать совместными усилиями.
Схема работы:
1. В чате всплывает уведомлении о начале регистрации на ивент. 
2. Игрок вводит команду в чат для принятия участия.
3. Игроки делятся на 2 разные команды ( Можно сделать подствеку игрокам как при ГвГ)
4. Игроки автоматически телепортируются в какую нибудь небольшую локацию.
5. N- кол-во времени идет драка, победитель команда которая убила больше всего человек.
6. Выдача награды победившим.

Большинство логики можно будет скорее всего взять из ГвГ функций.

**Comment by goodwinxp** [comment]
@StyleBlack, я правильно понял что хочется что-то типо такого?
https://www.youtube.com/watch?v=45mhDOjWzUo
https://www.youtube.com/watch?v=qp_KpUgFOek
или такое?
https://www.youtube.com/watch?v=LB0O4VRHpd4
https://www.youtube.com/watch?v=qp_KpUgFOek
https://www.youtube.com/watch?v=DOnxhkCCxic

**Comment by noargsconstruct** [comment]
Да, что-то типо. Не знал, что уже занимаются таким. Первое я так понял аналог ивента LastHero в l2.

**Comment by goodwinxp** [comment]
@StyleBlack, это демонстрация приватных аддонов от dev-corp для его защиты
